bin/citrea/provers/sp1/guest-bitcoin/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[workspace]
+
+[package]
+name = "citrea-sp1-guest-bitcoin"
+version = "0.5.0-rc.1"
+edition = "2021"
+resolver = "2"
+
+[dependencies]
+bitcoin-da = { path = "../../../../../crates/bitcoin-da", default-features = false }
+citrea-primitives = { path = "../../../../../crates/primitives" }
+citrea-sp1 = { path = "../../../../../crates/sp1", default-features = false }
+citrea-stf = { path = "../../../../../crates/citrea-stf" }
+sov-modules-api = { path = "../../../../../crates/sovereign-sdk/module-system/sov-modules-api", default-features = false }
+sov-modules-stf-blueprint = { path = "../../../../../crates/sovereign-sdk/module-system/sov-modules-stf-blueprint" }
+sov-rollup-interface = { path = "../../../../../crates/sovereign-sdk/rollup-interface" }
+sov-state = { path = "../../../../../crates/sovereign-sdk/module-system/sov-state" }
+sp1-zkvm = { git = "https://github.com/succinctlabs/sp1", tag = "v3.0.0-rc3", default-features = false, features = ["lib"] }
+
+# Have to put this here to enable features for the patch crate even though we don't use this crate explicitly
+curve25519-dalek = { version = "4.1.3", default-features = true }
+
+[patch.crates-io]
+# sha2-v0-9-8 = { git = "https://github.com/sp1-patches/RustCrypto-hashes", package = "sha2", branch = "patch-sha2-v0.9.8" }
```

Comment:
Please put down a comment why we don't use these patches

Reply:
done